packages/datatrak-web/src/api/queries/useSurveys.ts

```diff
@@ -17,4 +18,9 @@ export const useSurveys = () => {
         },
       }),
   );
+  // Filter the surveys by the selected country name
+  return {
+    isLoading: isLoading || !isFetched,
+    surveys: surveys?.filter((survey: any) => survey.countryNames?.includes(selectedCountryName)),
```

Comment:
This is an optional suggestion but I came across the concept of [react-query selectors](https://tkdodo.eu/blog/react-query-data-transformations#3-using-the-select-option) this morning and this would be a good candidate for it. The advantage is that it caches the transformation and also minimises re-renders.